backend/app/api/v1/endpoints/news.py

```
@@ -0,0 +1,98 @@
+    if not news:
+        raise HTTPException(status_code=404, detail="News subscription not found")
+    news = await crud_news.update(db, db_obj=news, obj_in=news_in)
+    return news
+
+
+@router.delete("/{news_id}", response_model=NewsSubscription)
+async def delete_news_subscription(
+    *,
+    db: deps.SessionDep,
+    news_id: int,
+) -> Any:
```

Ruff (0.14.7)

Dynamically typed expressions (typing.Any) are disallowed in `delete_news_subscription`

(ANN401)